src/app/app-menu.ts

```diff
@@ -204,81 +204,93 @@ export class AppMenu {
+                label: i18n.t('Always on Top')(),
+                type: 'checkbox',
+            },
+            {
+                checked: minimizeOnClose,
+                click: async (item) => {
+                    minimizeOnClose = item.checked;
+                    await config.updateUserConfig({ minimizeOnClose });
                 },
-                {
-                    checked: bringToFront,
-                    click: async (item) => {
-                        bringToFront = item.checked;
-                        await config.updateUserConfig({ bringToFront });
-                    },
-                    label: isWindowsOS
-                        ? i18n.t('Flash Notification in Taskbar')()
-                        : i18n.t('Bring to Front on Notifications')(),
-                    type: 'checkbox',
+                label: i18n.t('Minimize on Close')(),
+                type: 'checkbox',
+            },
+            {
+                checked: bringToFront,
+                click: async (item) => {
+                    bringToFront = item.checked;
+                    await config.updateUserConfig({ bringToFront });
                 },
-                this.buildSeparator(),
-                {
-                    checked: memoryRefresh,
-                    click: async (item) => {
-                        memoryRefresh = item.checked;
-                        await config.updateUserConfig({ memoryRefresh });
-                    },
-                    label: i18n.t('Refresh app when idle')(),
-                    type: 'checkbox',
+                label: isWindowsOS
+                    ? i18n.t('Flash Notification in Taskbar')()
+                    : i18n.t('Bring to Front on Notifications')(),
+                type: 'checkbox',
+            },
+            this.buildSeparator(),
+            {
+                checked: memoryRefresh,
+                click: async (item) => {
+                    memoryRefresh = item.checked;
+                    await config.updateUserConfig({ memoryRefresh });
                 },
-                {
-                    click: (_item, focusedWindow) => {
-                        if (focusedWindow && !focusedWindow.isDestroyed()) {
-                            const defaultSession = session.defaultSession;
-                            if (defaultSession) {
-                                defaultSession.clearCache(() => {
-                                    focusedWindow.reload();
-                                });
-                            }
+                label: i18n.t('Refresh app when idle')(),
+                type: 'checkbox',
+            },
+            {
+                click: (_item, focusedWindow) => {
+                    if (focusedWindow && !focusedWindow.isDestroyed()) {
+                        const defaultSession = session.defaultSession;
+                        if (defaultSession) {
+                            defaultSession.clearCache(() => {
```

Comment:
can we use async/await here?

Reply:
Current/latest electron versions still don't support

Reply:
No promises huh? Ok.